docs/operation/volume_management/volume_creation_deletion_cli.rst

```diff
@@ -0,0 +1,83 @@
+Volume Management using the CLI
+===============================
+
+To use persistent storage in a MetalK8s cluster, one needs to create Volume
```

Comment:
Let's be consistent in our highlighting of keywords (StorageClass, Volume, ...) with respect to the StorageClass documentation just above. Also, shouldn't these be `terms` leading to glossary entries?

Reply:
Yes, they could be terms leading to the glossary entries. But we do not have glossary terms for the following above mentioned words.

Maybe we could track this in another issue